charts/kof-mothership/values.yaml

```
@@ -130,6 +130,19 @@ grafana:
   # -- Version of Grafana to use.
   version: "10.4.7"
 
+  
+  dashboard:
+    # -- Datasource configuration
+    datasource:
+      # -- Regex pattern to filter datasources.
+      regex: "/promxy/"
+
+    # -- Filters configuration
+    filters:
+      # -- Filters data by cluster name.
+      clusterName: mothership
```

This results in too verbose table in README.

It makes sense to document a leaf or a group of leafs as a whole, but no need to document all intermediate branches:
```suggestion
    datasource:
      # -- Regex pattern to filter datasources.
      regex: "/promxy/"

    # -- Values of filters to apply.
    filters:
      clusterName: mothership
```